src/shell-window-tracker.c

```diff
@@ -434,6 +513,16 @@ get_app_for_window (ShellWindowTracker    *tracker,
   /* Side components don't have an associated app */
   if (g_strcmp0 (meta_window_get_role (window), SIDE_COMPONENT_ROLE) == 0)
     return NULL;
+  if (g_strcmp0 (meta_window_get_flatpak_id (window), BUILDER_WINDOW) == 0)
+  {
+    if (tracker->coding_app) {
+      g_hash_table_insert (tracker->builder_to_app, window, tracker->coding_app);
+      tracker->coding_app = NULL;
```

Comment:
So that handling in there prevents from appearing in places like the navigation bar or alt-tab. That was what I was after in the shell-window-tracker en gros.